Nodes/11 Image Modifier (Post Process)/Greyscale Halftone.clbnode.presets.json

```diff
@@ -17,11 +43,11 @@
                                         "Min": 0.0,
                                         "Has max bound": false,
                                         "Max": 1.0,
-                                        "Drag speed": 0.10000000149011612,
+                                        "Drag speed": 0.009999999776482582,
```

Comment:
I don't think this change was intentional, please discard it. Or if it is intentional, please explain why.